src/theme/theme.ts

```diff
@@ -36,6 +36,7 @@ const theme = createTheme({
         },
         divider: lightBlue[300]
     },
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
The new version of MUI introduces some typescript warnings and I'm confident it won't be an error type, so feel like it's fine to ignore